src/test/suite/stubDemoTutorial.test.ts

```diff
@@ -95,12 +98,14 @@ async function validateTerminalResponse(terminalName : string, terminalText : st
 	if (term) {
 		console.log(`-current terminal = ${term?.name}`);
 		await sendTerminalText(terminalName, terminalText);
-		await delay(delayTime);
-		const term2 = focusOnNamedTerminal(terminalName);
-		await delay(delayTime);
-		let result = await getTerminalOutput(terminalName);
-		console.log(`-validateTerminalResponse terminal output = ${result}`);
-		expect(result).to.include(terminalResponse);
+
+		const resultValue = await waitUntil(async () => {
+			focusOnNamedTerminal(terminalName);
+			const result = await getTerminalOutput(terminalName);
+			console.log(`-validateSimpleTerminalResponse terminal output = ${result}`);
+			return result.includes(terminalResponse);
+		}, 5000);
+		expect(resultValue).to.be.true;
```

Comment:
this assertion is no more used. if the `result` doesn't include the `terminalResponse` is not found, there is a timeout exception